resources/package.rb

```diff
@@ -87,7 +88,8 @@
         ignore_failure true
       end
     when 'mpkg', 'pkg'
-      execute "installation_file=$(ls '/Volumes/#{volumes_dir}' | grep '.#{new_resource.type}$') && sudo installer -pkg \"/Volumes/#{volumes_dir}/$installation_file\" -target /" do
+      allow_untrusted_cmd = allow_untrusted ? '-allowUntrusted' : ''
```

Comment:
It would make a bit more sense here if you just had a string called cmd with the previous logic to build the install string. Then you can append the -allowUntrusted  to it if allow_untrusted is set. Also that needs to be new_resource.allow_untrusted or it will break in Chef 14

Reply:
Good suggestion! I fixed this up. Still waiting on support@travis-ci.com to look at the "Abuse detected" ci builds. if you have any tips, I would appreciate. Thanks!